isis/appdata/import/PDS3/MroHirise.tpl

```diff
@@ -0,0 +1,99 @@
+{% extends "img_base.tpl" %}
+  {% block instrument %}
+    SpacecraftName                  = "{{INSTRUMENT_HOST_NAME.Value}}"
+    InstrumentId                    = {{INSTRUMENT_ID.Value}}
+    TargetName                      = {{TARGET_NAME.Value}}
+    StartTime                       = {{TIME_PARAMETERS.START_TIME.Value}}
+    StopTime                        = {{TIME_PARAMETERS.STOP_TIME.Value}}
+    ObservationStartCount           = {{TIME_PARAMETERS.MRO_OBSERVATION_START_COUNT.Value}}
+    SpacecraftClockStopCount        = {{TIME_PARAMETERS.SPACECRAFT_CLOCK_STOP_COUNT.Value}}
+    ReadoutStartCount               = {{TIME_PARAMETERS.MRO_READOUT_START_COUNT.Value}}
+    CalibrationStartTime            = {{TIME_PARAMETERS.MRO_CALIBRATION_START_TIME.Value}}
+    CalibrationStartCount           = {{TIME_PARAMETERS.MRO_CALIBRATION_START_COUNT.Value}}
+    MissionPhaseName                = "{{MISSION_PHASE_NAME.Value}}"
+    LineExposureDuration            = {{INSTRUMENT_SETTING_PARAMETERS.MRO_LINE_EXPOSURE_DURATION.Value}}
+    ScanExposureDuration            = {{INSTRUMENT_SETTING_PARAMETERS.MRO_SCAN_EXPOSURE_DURATION.Value}}
+    DeltaLineTimerCount             = {{INSTRUMENT_SETTING_PARAMETERS.MRO_DELTA_LINE_TIMER_COUNT.Value}}
+    Summing                         = {{INSTRUMENT_SETTING_PARAMETERS.MRO_BINNING.Value}}
+    Tdi                             = {{INSTRUMENT_SETTING_PARAMETERS.MRO_TDI.Value}}
+    FocusPositionCount              = {{INSTRUMENT_SETTING_PARAMETERS.MRO_FOCUS_POSITION_COUNT.Value}}
+    PoweredCpmmFlag                 = ({{ join(INSTRUMENT_SETTING_PARAMETERS.MRO_POWERED_CPMM_FLAG.Value, ", ") }})
+    CpmmNumber                      = {{INSTRUMENT_SETTING_PARAMETERS.MRO_CPMM_NUMBER.Value}}
+    CcdId                           = {{INSTRUMENT_SETTING_PARAMETERS.MRO_CPMM_NUMBER.Value}}
+    ChannelNumber                   = {{INSTRUMENT_SETTING_PARAMETERS.MRO_CHANNEL_NUMBER.Value}}
+    LookupTableType                 = ({{ join(INSTRUMENT_SETTING_PARAMETERS.MRO_LOOKUP_CONVERSION_TABLE.Value, ", ") }})
+    LookupTableNumber               = {{INSTRUMENT_SETTING_PARAMETERS.MRO_LOOKUP_TABLE_NUMBER.Value}}
+    LookupTable                     = ({{ join(INSTRUMENT_SETTING_PARAMETERS.MRO_LOOKUP_CONVERSION_TABLE.Value, ", ")}})
```

Comment:
The old test data doesn't have the "LookupTable" key. Was hi2isis just using it but not transferring it to the label?

Reply:
I believe so, hi2isis is using `LookupTable` as a key in the `MroHiriseInstrument.trn` file.

Reply:
Did some digging and found that `LookupTable` is being used in hi2isis temporally and then getting removed, I'll take it out of the template.